cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app/templates.xml

```diff
@@ -11,6 +11,7 @@
     <templateExtension ref="${variables.entityName}Eto.java" mergeStrategy="javamerge"/>
     <templateExtension ref="${variables.entityName}SearchCriteriaTo.java" mergeStrategy="javamerge"/>    
     <templateExtension ref="AbstractSearchCriteriaTo.java" mergeStrategy="javamerge"/>
+    <templateExtension ref="ApplicationComposedKeyEntity.java" destinationPath="../api/src/main/java/${variables.rootPackage}/general/common/api/ApplicationComposedKeyEntity.java" mergeStrategy="javamerge"/>
```

Comment:
I am not very convinced that it is the best approach to hardwire and restrict the name this way.
The actual demand and requirement behind this is to give projects more freedom over the currently "fixed" `ApplicationPersistenceEntity`. Couldnt we just support `Application*Entity` as a general pattern so projects can have their own classes such as `ApplicationCustomerKeyEntity`, `ApplicationIbanBicKeyEntity`, etc.?